lib/minesweeper_2pl/boardcli.rb

```diff
@@ -0,0 +1,57 @@
+module Minesweeper_2pl
+  class BoardCli
```

Comment:
What is the difference between a "CLI" and a "Board CLI"? Can you describe their single responsibilities in a sentence? Does the phrase "Board Command Line Interface" make sense when you say it out loud?